go-controller/pkg/retry/obj_retry.go

```diff
@@ -755,3 +761,40 @@ func (r *RetryFramework) WatchResourceFiltered(namespaceForFilteredHandler strin
 
 	return handler, nil
 }
+
+// getPendingPodsForNetwork returns all pods that are in the Pending state
+func getPendingPodsForNetwork(kubeClient kube.InterfaceOVN) ([]*kapi.Pod, error) {
```

Comment:
`getPendingPodsForNetwork` mighty be a tad misleading. You didn't like just `getPendingPods`?